fxa-auth-db-server/docs/API.md

```diff
@@ -667,7 +668,8 @@ curl \
       "type": "mobile"
       "createdAt": 1437992394186,
       "callbackURL": "https://updates.push.services.mozilla.com/update/abcdef01234567890abcdefabcdef01234567890abcdef",
-      "callbackPublicKey": "468601214f60f4828b6cd5d51d9d99d212e7c73657978955f0f5a5b7e2fa1370"
+      "callbackPublicKey": "BCp93zru09_hab2Bg37LpTNG__Pw6eMPEP2hrQpwuytoj3h4chXpGc-3qqdKyqjuvAiEupsnOd_RLyc7erJHWgA",
```

Comment:
do we have docs that help new contributors / devs understand the format of `callbackPublicKey` and `callbackAuthKey` somewhere here?